docs/workflows.md

```diff
@@ -78,7 +84,11 @@ Set `Workflow permissions` to `Read and write permissions`, then hit `Save`:
 
 ### Set up itch.io upload
 
-Create your itch.io page with `Kind of project = HTML`, and double-check that the [`ITCH_TARGET` variable](#set-up-variables) you've set up earlier matches your itch.io page.
+Create your itch.io page with the same name as is used in the `ITCH_TARGET` variable in [release.yaml](../.github/workflows/release.yaml).
+By default, this is the same as the project name.
+Also set `Kind of project` to `HTML`.
+
+After creation, you'll see a warning saying "There was a problem loading your project: No file provided to embed". Don't worry about it, we will fix that in a moment.
```

Comment:
```suggestion
On your project page, you'll see a warning saying "There was a problem loading your project: No file provided to embed". This is because we haven't uploaded a release yet.
```

Reply:
Not a fan of the second part, because it implies we will fix that right now. In fact, we first do the credential dance.

Reply:
Counteroffer: [`2ccdaa0` (#163)](https://github.com/TheBevyFlock/bevy_quickstart/pull/163/commits/2ccdaa045b6d2bf6f40ecf708dc8e8531431910d)